akka-actor-tests/src/test/scala/akka/routing/MetricsBasedResizerSpec.scala

```diff
@@ -244,8 +244,8 @@ class MetricsBasedResizerSpec extends AkkaSpec(ResizerSpec.config) with DefaultT
 
       resizer.reportMessageCount(router.routees, router.msgs.size)
 
-      val after = LocalDateTime.now
-      resizer.performanceLog(2).toMillis shouldBe (java.time.Duration.between(before, after).toMillis / 2 +- 1)
+      val after = System.nanoTime()
+      resizer.performanceLog(2).toMillis shouldBe ((before - after).nanos.toMillis / 2 +- 1)
```

Comment:
ok, but what happens if there is some unexpected pause somewhere?
those CI machines are very sluggish